osx/KA-Lite/KA-Lite/AppDelegate.m

```diff
@@ -1048,7 +1041,7 @@ - (void)discardPreferences {
 - (void)startKaliteTimer {
     // TODO(cpauya): Use initWithFireDate of NSTimer instance.
     // TODO(amodia): Check if kalite environment variables change.
-    [NSTimer scheduledTimerWithTimeInterval:5.0
+    [NSTimer scheduledTimerWithTimeInterval:08.0
```

Comment:
Why did we increase the delay to 8 seconds?  Is that why we removed the "delay" codes above?